packages/application-components/src/components/dialogs/form-dialog/form-dialog.tsx

```diff
@@ -31,6 +31,7 @@ export type TFormDialogProps = {
   dataAttributesPrimaryButton?: { [key: string]: string };
   getParentSelector?: () => HTMLElement;
   iconLeftSecondaryButton?: ReactElement;
+  leftAlignedFooterContent?: ReactNode;
```

Comment:
Can we name this simply `footerContent`?

It will always be left aligned to the buttons. It also gives more flexibility in case the layout changes in the future.

Reply:
I think it's due to the iconLeftSecondaryButton existing also to be consistent. I also would like footerElement more cause we're passing a React element.

Reply:
Yes @tdeekens that would also work. I think both names suggested cover the scope of the prop and its use case. 
I could change it if you want. 
Thanks!

Reply:
@ddouglasz Maybe check in App Kit and UI Kit if there are similar cases and see what kind of naming we are using there.

Reply:
Yes, thank you for the suggestion. We have cases like these mostly with "content". Like the MaintenancePageLayout with a prop of `bodyContent?: ReactNode;`